### DISCUSSION TITLE
Cannot boot up High Sierra, stuck in UEFI/BIOS

### DISCUSSION THREAD

**Discussion by ghost** [utmapp/UTM#3956] [UTM Help (macOS)] [answered]
When I try to boot up High Sierra (below,) it boots up into the UEFI shell, and seems there is no clear way to boot up the actual OS.

https://user-images.githubusercontent.com/82232790/166119400-c06ab6f8-4412-408b-8939-98d5ec605c02.mp4

Am I missing anything?

Currently using 2020 Macbook Air with M1

**Comment by 3DCoded** [comment, marked as answer]
macOS High Sierra is an x86_64 os, not arm64. You would need to make the VM x86_64 emulation.

**Reply by ghost** [reply]
I'm new to UTM, how do I set it to x86_64?

**Reply by 3DCoded** [reply]
In VM settings, go to system and in architecture select x86_64.

**Reply by ghost** [reply]
Where is VM Settings?
As in, how do you get to it?

**Reply by 3DCoded** [reply]
At the upper right of the UTM window, there are a bunch of switches. Click on them to go to the VM settings.

**Reply by ghost** [reply]
Thank you!

**Reply by ghost** [reply]
<img width="261" alt="Screen Shot 2022-04-30 at 3 58 05 PM" src="https://user-images.githubusercontent.com/82232790/166120814-5301e46f-9bf9-4b0a-ba3d-f614d1a169ef.png">
What's happening currently?

**Reply by 3DCoded** [reply]
I'm not sure. I haven't gotten UTM to boot macOS other than macOS 12 yet.

**Reply by ghost** [reply]
It seems to be a problem with the UEFI boot. When I disable it, the error does not happen.

**Reply by ghost** [reply]
Trying to start it up, get this
<img width="471" alt="Screen Shot 2022-04-30 at 4 12 06 PM" src="https://user-images.githubusercontent.com/82232790/166121160-e1f64188-f416-46f7-9f03-741abadd1964.png">
:

**Reply by ghost** [reply]
Is it ok that I'm using a DMG?

**Reply by 3DCoded** [reply]
I'm not too experienced with runnign macOS on UTM. If you want to run macOS on UTM, I recommend you use the official method to run macOS Monterey.

**Reply by ghost** [reply]
I see. However, I am trying to use an older version of MacOS which supports 32 bit applications because I am trying to run 32 bit games from the App Store.
